packages/components/src/core/Autocomplete/index.tsx

```diff
@@ -147,13 +147,27 @@ const Autocomplete = <
                */
               ...params.InputProps.ref,
               endAdornment: (
-                <InputAdornment position="end">
+                <StyledInputAdornment position="end">
```

Comment:
I replaced the default HTML clear icon for search inputs with an SDS xMark icon as an input adornment at the end of the input.

Reply:
feels like it's worth adding this comment in the code!

Reply:
Ahh, sure :D